Kitodo/src/main/java/org/kitodo/production/helper/SearchResultGeneration.java

```diff
@@ -60,38 +56,34 @@ public HSSFWorkbook getResult() {
         return getWorkbook();
     }
 
-    private List<ProcessDTO> getResultsWithFilter() {
-        List<ProcessDTO> processDTOS = new ArrayList<>();
-        try {
-            processDTOS = ServiceManager.getProcessService().findByQuery(getQueryForFilter(ObjectType.PROCESS),
-                ServiceManager.getProcessService().sortById(SortOrder.ASC), true);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
-        }
-
-        return processDTOS;
+    private List<Process> getResultsWithFilter() {
+        List<Process> processInterfaces;
+        processInterfaces = ServiceManager.getProcessService().getByQuery(getQueryForFilter(Process.class)
+                    + " ORDER BY id ASC");
+        return processInterfaces;
     }
 
     /**
      * Gets the query with filters.
      *
-     * @param objectType Type of object that should be filtered
-     * @return A BoolQueryBuilder
+     * @param objectType
+     *            Type of object that should be filtered
+     * @return A query
      */
-    public BoolQueryBuilder getQueryForFilter(ObjectType objectType) {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-
-        try {
-            query = ServiceManager.getFilterService().queryBuilder(this.filter, objectType, false, false);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
+    public String getQueryForFilter(Class<? extends BaseBean> objectType) {
+        String query = "FROM Process AS process";
```

Comment:
I'm hoping that the created query is a Hibernate query in HQL and not a raw SQL query as raw SQL queries are not portable.

Reply:
Yes, it are all HQL queries